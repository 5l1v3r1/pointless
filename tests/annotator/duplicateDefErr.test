

y = 123
y = 4