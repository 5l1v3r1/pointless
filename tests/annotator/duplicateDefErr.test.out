-------------------------------------------------------------------------------
Name Error:

Duplicate definition for name 'y'
-------------------------------------------------------------------------------

At (line 4) (column 1) in 'tests/annotator/duplicateDefErr.test'
y = 4
^
